woocommerce/woocommerce-android
# Show snackbar when there is an error syncing Inbox

Closes: #6279 


### Description
Display a snackbar with the message: "Error syncing inbox" when any of the following requests fails inside Inbox screen: 
- When fetching inbox notes from API fails
- When dismissing one note fails
- When dismissing all notes fails 
- Swipe to refresh

### Testing instructions
 Remove connectivity and check snackbar with error "Error syncing Inbox" is displayed in following cases:
- Open Inbox
- Press dimiss over inbox note (if there are no inbox notes cached then enable connectivity to properly load Inbox and then retry this tests with no connectivity)
- Press dismiss all notes. Note that this has an "optimistic" approach. So local DB will be cleared even if the API requests fails. 

### Images/gif


https://user-images.githubusercontent.com/2663464/163812216-b1b88696-817a-4b37-9f72-b710afc2c8cb.mp4
